pkg/cache/v3/snapshot.go

```diff
@@ -194,3 +226,52 @@ func (s *Snapshot) GetVersion(typeURL string) string {
 	}
 	return s.Resources[typ].Version
 }
+
+// GetVersionMap will return the internal version map of the currently applied snapshot.
+func (s *Snapshot) GetVersionMap() map[string]map[string]string {
+	return s.VersionMap
+}
+
+// ConstructVersionMap will construct a verison map off the current state of a snapshot
```

Comment:
`baed on`/`based of`

Reply:
Oops sorry, thought I fixed that already.